### ISSUE TITLE
[Drupal] Clean up tech debt from Connect With Us

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#19794]
## Description

After the Connect With Us work is completed and released, clean-up of tech debt should occur (see AC below).

## Notes
No Connect With Us data in Outreach and Events so we are opting to remove the fieldset from the Offices content type.

## Question(s)
From Jill in comments below: the one FE piece here is re: removing the Offices version of Connect With Us fields from CLPs/BHLPs, but I'm pretty sure in your FE PR, you just swapped that at the time, so there aren't references left from Offices in the FE, right? If that's true, we can remove that AC and this becomes a Drupal-only ticket.

Note: removed FE AC.

### Drupal
- [x] Archive VHA, VBA, NCA, VACO Office nodes
- [ ] Remove Connect with Us fields from Offices content type (these were hidden via PR #19970 but need to be removed) 
- [ ] Remove Office-powered Connect with Us Fields from CLP and BHLP content types
- [ ] Remove entity reference view that is no longer needed
- [ ] Remove Node View display mode that is no longer needed - core.entity_view_display.node.office.connect_with_us.yml
- [ ] Remove this [Twig config](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/docroot/themes/custom/vagovclaro/templates/node/node--office--connect-with-us.html.twig) or update it to apply to new Connect With Us fields?
- [x] Update [CLP Cypress Test](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/tests/cypress/integration/features/content_type/campaign_landing_page/clp_connect_with_us.feature) to reference new field 
- [ ] This [BHLP Cypress test](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/tests/cypress/integration/features/content_type/landing_page.feature) doesn't reference Connect With Us so should be fine as is
- [ ] The [Office Cypress test](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/tests/cypress/integration/features/content_type/office.feature) doesn't reference Connect With Us so should be fine as is

### Documentation
- [ ] Add Content Model Document for Connect with us block
- [ ] Update KB articles

**Comment by davidmpickett** [comment]
I just started a basic PR to Hide the deprecated fields https://github.com/department-of-veterans-affairs/va.gov-cms/pull/19970

This should be able to merge pretty quickly and at least imporve the editor experience by not having two nearly identical options displayed. Clears the way for fully removing fields later.

**Comment by jilladams** [comment]
@davidmpickett the one FE piece here is re: removing the Offices version of Connect With Us fields from CLPs/BHLPs, but I'm pretty sure in your FE PR, you just swapped that at the time, so there aren't references left from Offices in the FE, right? If that's true, we can remove that AC and this becomes a Drupal-only ticket.

**Comment by jilladams** [comment]
Also I'm curious: @davidmpickett is the scope of this ticket in your wheelhouse, or bleeds past your Drupal edges? IF it's fully in your wheelhouse, feel free to take a stab at points.

**Comment by davidmpickett** [comment]
> Also I'm curious: [@davidmpickett](https://github.com/davidmpickett) is the scope of this ticket in your wheelhouse, or bleeds past your Drupal edges? IF it's fully in your wheelhouse, feel free to take a stab at points.

I've never removed fields from Drupal, so that would be a stretch and my points for this would likely be higher than @omahane or @dsasser. I think I also need to focus on R&S prototype so I don't think I can commit to fully running this ticket

**Comment by davidmpickett** [comment]
> [@davidmpickett](https://github.com/davidmpickett) the one FE piece here is re: removing the Offices version of Connect With Us fields from CLPs/BHLPs, but I'm pretty sure in your FE PR, you just swapped that at the time, so there aren't references left from Offices in the FE, right? If that's true, we can remove that AC and this becomes a Drupal-only ticket.

Yeah, I'm pretty sure all FE references to Office-powered Connect With Us fields are gone with my PR so this is probably Drupal-only

**Comment by davidmpickett** [comment]
Confirmed that #19970 has deployed and old fields are now hidden on Prod CMS

**Comment by jilladams** [comment]
This has been deployed now, and I've double checked everything in Prod that I can grok from the ACs / FE. There are a couple things I'm not positive how to verify: 
Remove Node View display mode that is no longer needed - core.entity_view_display.node.office.connect_with_us.yml

Remove this [Twig config](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/docroot/themes/custom/vagovclaro/templates/node/node--office--connect-with-us.html.twig) or update it to apply to new Connect With Us fields?

**Comment by omahane** [comment]
Agreed that you can't verify the removed Twig or entity reference view.

I looked at the [View Modes](https://prod.cms.va.gov/admin/structure/display-modes/view) and see none for Connect with us.
